apheleia.el

```diff
@@ -6,9 +6,9 @@
 ;; Created: 7 Jul 2019
 ;; Homepage: https://github.com/raxod502/apheleia
 ;; Keywords: tools
-;; Package-Requires: ((emacs "26"))
+;; Package-Requires: ((emacs "26") (editorconfig "0.9"))
 ;; SPDX-License-Identifier: MIT
-;; Version: 3.2
+;; Version: 3.3
```

Comment:
I think we only want to bump the version here when we actually release the new version.